include/worker.hpp

```diff
@@ -356,5 +401,35 @@ class Worker
 
     // Path to config JSON if applicable.
     std::string& m_configJsonPath;
+
+    // To perform backup and restore
+    bool m_performBackupAndRestore;
```

Comment:
Initialize here itself and re-order functions and variables.

Reply:
Done